common/cpp/src/google_smart_card_common/external_logs_printer.cc

```diff
@@ -44,6 +45,14 @@ void PrintExternalLogMessage(const ExternalLogMessageData& message_data) {
 
 }  // namespace
 
+template <>
+StructValueDescriptor<ExternalLogMessageData>::Description
+StructValueDescriptor<ExternalLogMessageData>::GetDescription() {
+  return Describe("ExternalLogMessageData")
+      .WithField(&ExternalLogMessageData::formatted_log_message,
```

Comment:
Question: Maybe I should introduce a short alias like `T` that would allow to shorten the descriptions, to something like:

```
  .WithField(&T::formatted_log_message, "formatted_log_message");
```

... or does the clarity of the current approach outweigh its verbosity?